SmartPlaces.Facilities/lib/IngestionManager.Mapped/test/MappedGraphIngestionProcessorTests.cs

```diff
@@ -0,0 +1,57 @@
+// -----------------------------------------------------------------------
+// <copyright file="MappedGraphIngestionProcessorTests.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.SmartPlaces.Facilities.IngestionManagerManager.Mapped.Test
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights;
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.SmartPlaces.Facilities.IngestionManager;
+    using Microsoft.SmartPlaces.Facilities.IngestionManager.Interfaces;
+    using Microsoft.SmartPlaces.Facilities.IngestionManager.Mapped;
+    using Microsoft.SmartPlaces.Facilities.OntologyMapper;
+    using Moq;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class MappedGraphIngestionProcessorTests
+    {
+#pragma warning disable IDE0052 // Remove unread private members
+        private readonly ITestOutputHelper output;
```

Comment:
Does this actually get used in the test? I see no reference to it in the code other than the constructor

Reply:
Shouldn't there be an `output.WriteLine` somewhere in this file?